### PR TITLE
Add Phenix and merged with bioconda:master

### PR DESCRIPTION
**Pull request by thanhleviet**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/phenix/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{% set name = "PHEnix" %}
+{% set version = "1-4-1a" %}
+{% set hash = "3153ebf851b0418f81afe0e3c1ec2b52a43067c134020d83a7d0faf351583741" %}
+
+package:
+  name: {{ name | lower }}
+  version: "{{ version | replace('-', '_') }}"
```
Comment (bgruening) [inline recipes/phenix/meta.yaml]: can this be a dot?
Why is this such a strange version number :)

Comment (thanhleviet) [reply recipes/phenix/meta.yaml]: I replaced "-" by ".",  but the version number I kept as same as the original release at here https://github.com/phe-bioinformatics/PHEnix/releases  :).

**Comment by thanhleviet** [comment]
@bgruening Is it OK for merging to bioconda master now? Thanks

**Comment by bgruening** [comment]
Thanks. If you update the recipe, please try to remove the build.sh file, include the line as build/script and I think many of the build dependencies are not needed. But lets get this in for now :)

**Comment by thanhleviet** [comment]
Thanks for guidelines, I will update in future  version :)

**Pull request merged**